README.md

```diff
@@ -288,6 +289,15 @@ on several packages in parallel via git. Details can be found
 > You can see an example of such usage [here](https://github.com/stefan-hoeck/idris2-pack-db/blob/bcc8dc61706c73361bb1e6e18dd1b0c5981f0e18/collections/HEAD.toml#L297).
 > Technical details can be found [here](https://github.com/stefan-hoeck/idris2-pack/issues/256#issuecomment-1689305587).
 
+## Upgrade Idris2 Compiler
+
+In order to upgrade to the latest Idris2 compiler use
```

Comment:
This will not only update the Idris compiler but switch to the latest pack collection. If we add this section to the README, we should correctly explain what this means.